src/pages/todo/reducers.js

```diff
@@ -1,10 +1,16 @@
 import { reduceApi } from '../../common/api'
 
 import {
-  LIST_TODOS
+  LIST_TODOS,
+  DETAIL_TODOS
 } from './actions';
 
 export default function todoReducer(state = {todos: []}, action) {
-  state = reduceApi(state, action, LIST_TODOS, (data) => { return {todos: data}; });
+  if (Object.values(LIST_TODOS).indexOf(action.type)  !== -1) {
```

Comment:
@vinhkq (`Object.values(LIST_TODOS)` should be `Object.values(LIST_TODOS + '_')` to avoid conflicts.)
But please define the check function in common/api.js: 
```
export function getBaseType(action) {
  ...
  return baseType;
}
```
and then in pages/todo/reducers.js:
```
switch(getBaseType(action)) {
  case LIST_TODOS: ...
}
```